egs/wsj/s5/steps/paste_feats.sh

```diff
@@ -69,12 +76,16 @@ $cmd JOB=1:$nj $logdir/append.JOB.log \
    copy-feats --compress=$compress ark:- \
     ark,scp:$ark_dir/pasted_$name.JOB.ark,$ark_dir/pasted_$name.JOB.scp || exit 1;
 
+# create the cmvn scp file
+if [ "$create_cmvn_scp" == "true" ]; then
+  paste-feats --length-tolerance=0 $cmvn_src_args ark,scp:$ark_dir/cmvn.ark,$data/cmvn.scp
```

Comment:
this wouldn't work right because the CMVN stats have an extra dimension for the counts.  I think it would be safer to just recompute the CMVN stats.